boringtun/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "boringtun"
+description = "an implementation of the WireGuard® protocol designed for portability and speed"
+version = "0.5.2"
+authors = ["Noah Kennedy <nkennedy@cloudflare.com>", "Andy Grover <agrover@cloudflare.com>", "Jeff Hiner <jhiner@cloudflare.com>"]
+license = "BSD-3-Clause"
+documentation = "https://docs.rs/boringtun/0.5.2/boringtun/"
+edition = "2018"
+
+[features]
+jni-bindings = ["ffi-bindings", "jni"]
+ffi-bindings = []
+
+[dependencies]
+base64 = "0.13"
+hex = "0.4"
+untrusted = "0.9.0"
+libc = "0.2"
```

Comment:
this probably should be optional, and included only with ffi-bindings

Reply:
Made it optional on ffi-bindings and it does not compile. This crate extensively used by `boringtun/src/device/udp_unix.rs`